### PR TITLE
refs #74 support Turnip

### PR DESCRIPTION
**Pull request by oxoofo**
#74

### CODE REVIEW

**Comment by bhuga** [comment]
@oxoofo, this looks about right. Can you add a test that uses turnip?

**Comment by oxoofo** [comment]
@bhuga, Thank you for your comment. I’m sorry about PR without test...  I added tests.

May I remake my all commits squashing to 1 or 2 commits, and push forcefully here?

**Comment by bhuga** [comment]
> May I remake my all commits squashing to 1 or 2 commits, and push forcefully here?

If you prefer, yep!

File: test/samples/features/turnip_steps/global_steps.rb [hunk 1]
```diff
@@ -0,0 +1,19 @@
+step %(a) do
+  sleep 0.10
+end
+
+step %(b) do
+  sleep 0.25
```
Comment (bhuga) [inline test/samples/features/turnip_steps/global_steps.rb]: Why were these `sleep`s needed?

Comment (oxoofo) [inline test/samples/features/turnip_steps/global_steps.rb]: Thank you for your review. Sorry, these `sleep`s were needless, I think.
I accidentally left these when I wrote this test referring cucumber test.
I removed these. (7a92f59)

I also removed other needless codes I found in turnip.bats/setup,teardown

**Comment by oxoofo** [comment]
I remade my commits squashing, and pushed forcefully. (14240fc)

**Comment by koic** [comment]
@oxoofo I opened https://github.com/tmm1/test-queue/pull/120 that your commit base instead. So I'm sorry about this PR will be closed.

**Pull request closed without merging**